db/dbformat.h

```diff
@@ -324,11 +333,24 @@ class InternalKey {
     SetFrom(ParsedInternalKey(_user_key, s, t));
   }
 
+  void Set(const Slice& _user_key, SequenceNumber s, ValueType t,
```

Comment:
Better make it explicit via naming or comment about whether "_user_key" contains timestamp. Too bad we cannot use type to distinguish the two at this moment.

Reply:
Updated argument name.